src/Philter.Aura/Philter.Aura.Web/Program.cs

```diff
@@ -41,6 +45,7 @@
 );
 
 services.AddCoalesce<AuraDbContext>();
```

Comment:
```suggestion
services.Configure<TwilioOptions>(builder.Configuration.GetSection(TwilioOptions.Section));
```
Don't you need something like this in order to complete the IOptions pattern?

Reply:
Sorry... don't remove that line of code.... lol. My code suggestion is bad. I'll fix it.